app/views/pages/contributors.html.erb

```diff
@@ -80,4 +80,17 @@ We're always looking for ideas from everyone, including <strong>mental health ad
 			<i class="fa fa-globe fa-inline fa-left"></i><%= link_to('alexfa.net', 'http://alexfa.net/', class: "spacing") %>
 		</div>
 	</div>
-</div>
\ No newline at end of file
+	<div class="table_row">
+		<div class="table_cell" style="width: 200px; padding-right: 40px; padding-bottom: 40px; vertical-align: middle">
```

Comment:
Get rid of "padding-bottom: 40px" in your cell and add it to the row before yours! Other than that, looks good!